app/Services/MovieServices/CommentService.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Services\MovieServices;
+
+use App\Comment;
+use App\Movie;
+use App\User;
+
+class CommentService {
+    public function getCommentsForMovie ($movie)
+    {
+       $comments = $movie->comments;
+       return $this->attachUserToComment($comments);
+    }
+    public function createComment ($movieId, $content, $userId)
+    {
+      return Comment::create(['content'=>$content, 'movie_id'=>$movieId,'user_id'=>$userId]);
+    }
+    public function deleteComment($comment){
+        $comment->delete();
+        return $comment;
+    }
+    public function attachUserToComment ($comments)
```

Comment:
nonononono

Reply:
ovo ce ti za svaki komentar napraviti jos jedan query ka bazi, a to nikako ne zelis (nadam se :joy: )